compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -13245,7 +13245,7 @@ TR::Node *ificmpneSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier
       }
 
    if (node->getOpCodeValue() == TR::ificmpne)
-      intCompareNarrower(node, s, TR::ifsucmpne, TR::ifscmpne, TR::ifbcmpne);
+      intCompareNarrower(node, s, TR::ifscmpne, TR::ifscmpne, TR::ifbcmpne); //ushortOp is the same as shortOp in this case
```

Comment:
not sure why this needs a comment - the others don't

Reply:
Not sure why this comment is there. I don't think I wrote that comment. Do you think it is a good idea to just remove it as it does not give much useful information.

Reply:
looks like you added it, but regardless it is redundant and should be removed IMO.